board/Ladder.java

```diff
@@ -0,0 +1,23 @@
+package board;
+
+public class Ladder {
+	private int start;
+    private int end;
+
+    public Ladder(int startval, int endval){
+        start = startval;
```

Comment:
startVal instead of startval

You can also do the cleaner version:
this.start = start;